lsp/eslint.lua

```diff
@@ -172,11 +172,17 @@ return {
       end, eslint_config_files)
 
       for _, file in ipairs(flat_config_files) do
-        local found_files = vim.fs.find(function(name, path)
-          return name == file and not path:match('[/\\]node_modules[/\\]')
-        end, { path = root_dir, type = 'file', limit = 1 })
+        local found_files = vim.fn.globpath(root_dir, file, true, true)
 
-        if #found_files > 0 then
+        -- Filter out files inside node_modules
+        local filtered_files = {}
+        for _, found_file in ipairs(found_files) do
+          if string.find(found_file, '/node_modules/') == nil then
```

Comment:
globpath might return `\` slashes on windows?

Reply:
Argh something I'm not super familiar with :/ I guess in this case I should match with a regex? Or maybe just `node_modules` would be sufficient, it's a private dir name in Node for sure

Reply:
Just added the same pattern as before in fact as `:find` works as well